src/site/content/en/blog/bluetooth/index.md

```diff
@@ -145,12 +149,51 @@ navigator.bluetooth.requestDevice({
   filters: [{
     name: 'Francois robot'
   }],
-  optionalServices: ['battery_service']
+  optionalServices: ['battery_service'] // Required to access service later.
 })
 .then(device => { /* … */ })
 .catch(error => { console.error(error); });
 ```
 
+#### Manufacturer data filter
+
+It is also possible to request Bluetooth devices based on the manufacturer
+specific data being advertised with the `manufacturerData` filters key. This key
+is an array of objects with a mandatory [Bluetooth company identifier] key named
+`companyIdentifier`. You can also provide a data prefix that filters
+manufacturer data from Bluetooth devices that start with it. Note that you will
+also need to define the `optionalServices` key to be able to access some
+services. If you don't, you'll get an error later when trying to access them.
```

Comment:
Rather than "some services" I would say "any services not included in a service filter".

Reply:
Done